shotover-proxy/tests/test-configs/hotreload/docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+  services:
+    redis:
+      image: library/redis:5.0.9
+      ports:
+        - "6379:6379"
+      healthcheck:
```

Comment:
None of our other `docker-compose.yaml` files use a healthcheck, I dont think its really needed in an integration test scenario, can we remove it?